### PR TITLE
Add comments to normalizeData function

### PR DESCRIPTION
**Pull request by Nikos-Antonopoulos**
@XhmikosR This pull request is part of a project for a university assignment concerning construction quality and specifically good code documentation. There is no need to review this pull request. Feel free to close it or merge it :)

### CODE REVIEW

**Comment by GeoSot** [comment]
Hey Nikos. thank you for your Pr. You may remove the inline comments and keep only the function doc-block (8-13)

Also try to squash your commits into one and re-base the branch

**Comment by Nikos-Antonopoulos** [comment]
@GeoSot, i have applied your suggestions. Hopefully, everything is ok now!

File: js/src/dom/manipulator.js [hunk 1]
```diff
@@ -5,6 +5,13 @@
  * --------------------------------------------------------------------------
  */
 
+/**
+ * Returns the val in the correct primitive type.
+ *
+ * @param {String} val A value of any primitive type as a String
```
Comment (alpadev) [inline js/src/dom/manipulator.js:11]: Shouldn't string be lowercase?

**Comment by XhmikosR** [comment]
@Nikos-Antonopoulos this whole PR is a little moot in the sense that we don't use JSDoc comments anywhere else.

So, while I appreciate the effort, unless this is done across the codebase and enforced somehow, it doesn't make sense to proceed with this PR.

**Comment by Nikos-Antonopoulos** [comment]
@XhmikosR as I've already mentioned in the first comment, this PR was created for a university assignment and there is no need to review it, so feel free to close it! However, thank you very much for your time!

**Comment by XhmikosR** [comment]
Haha, sorry, I thought you actually wanted to land this. All good, then :)

**Pull request closed without merging**